dfttopif/drivers.py

```diff
@@ -213,10 +225,8 @@ def convert(files=[], **kwargs):
 
     if len(files) == 1:
         if os.path.isfile(files[0]):
-            return directory_to_pif(os.path.dirname(files[0]), **kwargs)
+            return files_to_pif(files, **kwargs)
         else:
-            return directory_to_pif(files[0], **kwargs)
+            raise ValueError('You need to pass at least one file!')
```

Comment:
shouldn't this be `return directory_to_pif(files[0], **kwargs)`?

Reply:
This is the only approval blocker

Reply:
Yea, it should